src/features/harborView/forms/berthForm/utils/utils.ts

```diff
@@ -16,3 +16,22 @@ export const getBerth = (berthData: INDIVIDUAL_BERTH | undefined): Berth | undef
     pier: berthData.berth.pier.properties?.identifier,
   };
 };
+
+export const isNumber = (value: string | undefined): boolean => {
+  if (value === undefined) return true; // required is a separate test
+  const numberRegex = RegExp('^-?\\d+([.,]\\d+)?$');
```

Comment:
One thing I forgot to note. don't we need a `new` operator?

Reply:
Yeah! I'll fix that one quick. Strange, it seems to work without it.